src/q7/radiation-ELEC2795/exam/2020/Janvier/All/radiation-ELEC2795-exam-2020-Janvier-All.tex

```diff
@@ -0,0 +1,109 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\usepackage{../../../../../../eplunits}
+
+\graphicspath{{img/}}
+
+\hypertitle{Radiation and communication systems}{7}{ELEC}{2795}{2020}{Janvier}{All}
+{Martin Braquet \and Sébastien Couvreur \and Oriane de Leuze}
+{Christophe Craeye, Danielle Janvier, Jérome Louveaux, Claude Oestges and Luc Vandendorpe}
+
+\section{(4,5 points)}
+A ground station is emitting a plane wave toward a satellite, with an elevation angle $\alpha_1=10^o$. The electric field is horizontally polarized. The atmosphere is modelled as two dielectric layers separated by a horizontal interface. The relative dielectric permittivity of the two layers is $\epsilon_1$ and $\epsilon_2 = 1.01\epsilon_1$ respectively.
+
+\begin{figure}[!h]
+    \centering
+    \includegraphics[scale=0.3]{Q1.png}
+\end{figure}
+
+\begin{enumerate}
+    \item (1 point) Calculate the elevation angle in layer 2 ($\alpha_2$ in the figure).
+    \item (2 point) Calculate the reflection coefficient $\Gamma$ and the transmission coefficient T of the electric field at the interface between the to layers.
+    \item (1,5 points) If the distance between the satellite and the ground station (Matera) is 40500 km and the frequency 7,8 GHz, calculate the free space losses + total tropospheric losses exceeded during 0.01\% of the time.
+\end{enumerate}
+(CCDF graph available with an attenuation of 9 dB for 0.01\% of the time at the angle and frequency related to this problem)
+
+\nosolution
+
+
+\section{(4,5 points)}
+
+Two parabolic dish antennas are facing each other to establish a horizontal link over a distance of 2 km. The frequency of operation is 5 GHz and the diameter of the dishes is 1 meter. Both antennas have an input impedance of 50 Ohm, while the generator impedance is $(100-j50)$ Ohm on the transmitting side and the receiver impedance is 50 Ohm on the receiving side. The noise figure of the receiver is 3 dB, the bandwidth of the signal is 5 MHz. The transmitting polarization is vertical, while the receiving polarization is circular. 10 milliWatt are available at the generator.
+\begin{enumerate}
+    \item Assuming in a first instance that the two antennas have the same polarization and that they are impedance matched, what is the signal-to-noise ratio on the receiver?
+    \item What does that SNR become with the impedance and polarization mismatches referred to above?
+    \item With which accuracy (in degrees) do the two antennas need to be pointed at each other to avoid a decrease of the signal by more than 3dB? (do the calculation assuming that one antenna is perfectly pointed and the other is not)
```

Comment:
```suggestion
    \item With which accuracy (in degrees) do the two antennas need to be pointed at each other to avoid a decrease of the signal by more than \SI{3}{dB}? (do the calculation assuming that one antenna is perfectly pointed and the other is not)
```